recognition/advanced_tree_search.py

```diff
@@ -817,7 +817,16 @@ class BuildGlobalCacheJob(rasr.RasrCommand, Job):
     Standalone job to create the global-cache for advanced-tree-search
     """
 
-    def __init__(self, crp, extra_config=None, extra_post_config=None):
+    __sis_hash_exclude__ = {"search_type": "advanced-tree-search", "recognizer_type": "recognizer"}
```

Comment:
no need for `__sis_hash_exclude__` for this Job, since it implements its own hash method that makes the hash only depend on the generated config and not the input arguments that are passed to the job.